packages/blocks/src/store/actions.js

```diff
@@ -37,7 +45,7 @@ const LEGACY_CATEGORY_MAPPING = {
  * Whether the argument is a function.
  *
  * @param {*} maybeFunc The argument to check.
- * @return {boolean} True if the argument is a function, false otherwise.
+ * @return {maybeFunc is Function} True if the argument is a function, false otherwise.
```

Comment:
nice. first time I've seen this in JSDoc

Reply:
@dmsnell I'm curious, these sorts of functions seem like they should be in a utils common package, rather than implemented in separate places. I've seen a lot of `isObject` functions stashed in multiple packages of Gutenberg.